core/config-manager/configurationmanager_test.go

```diff
@@ -254,6 +246,34 @@ ssl:
   certPwdFile:
 
 commonkey3 : filesource
+`)
+
+	loadbalanceConf := []byte(`
+--- 
+cse: 
+  loadbalance: 
+    backoff: 
```

Comment:
maybe that should be a concret service name

Reply:
I just gave that name backoff to verify that the inline tag can contain even names which may matched other tag

Do i need to change now or is that okey?